tests/js-units/node/state/maplayer.test.js

```diff
@@ -10,6 +10,7 @@ import { LayerIconSymbology, LayerSymbolsSymbology, SymbolIconSymbology } from '
 import { LayerGroupState, LayerVectorState, LayersAndGroupsCollection } from '../../../../assets/src/modules/state/Layer.js';
 
 import { MapLayerLoadStatus, MapGroupState, MapLayerState } from '../../../../assets/src/modules/state/MapLayer.js';
+import exp from 'constants';
```

Comment:
Seems not be used

Reply:
Ouch, I didn't notice, sorry. It seems to have been added automatically by my text editor.